src/main/java/com/fasterxml/jackson/databind/deser/std/MapDeserializer.java

```diff
@@ -196,6 +225,10 @@ public void setIgnorableProperties(Set<String> ignorable) {
                 null : ignorable;
     }
 
+    public void setIncludableProperties(Set<String> includable) {
+        _includableProperties = includable;
```

Comment:
Ideally would be great to avoid mutability; I realize there is existing `setIgnorableProperties` however, so maybe this can not be avoided? (called from `BasicDeserializerFactory`)

Reply:
If you are fine with it, I could add  a `MapDeserializer` constructor that already takes `included` and `ignored` as arguments, so we don't need the setter. I don't need then to add this setter and can mark the other as deprecated.

Reply:
That sounds reasonable: I suspect there is a reason for setter but cannot remember what exactly it was. Could be related to use of `MapDeserializer` for "any properties" handling or something...